Hospital.Infrastructure.InMemory/AppointmentInMemoryRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Hospital.Domain.Entities;
+using Hospital.Tests;
+using System.Data;
+
+namespace Hospital.Infrastructure.InMemory.Repositories;
```

Comment:
Неверный namespace. Не соответствует структуре папок.
```suggestion
namespace Hospital.Infrastructure.InMemory;
```